Library/Homebrew/cmd/test-bot.rb

```diff
@@ -425,6 +425,10 @@ def formula formula_name
       install_args = %w[--verbose]
       install_args << "--build-bottle" unless ARGV.include? "--no-bottle"
       install_args << "--HEAD" if ARGV.include? "--HEAD"
+      # Pass --devel or --HEAD to the test-bot in the event formulae lack stable. Supports devel-only/head-only.
+      # head-only should not have devel, but devel-only can have head. Stable can have all three.
+      install_args << "--devel" if formula.devel && !formula.stable?
+      install_args << "--HEAD" if formula.head && !formula.stable? && !formula.devel?
```

Comment:
Sure. I'll look into moving this element into the block below.

Not 100% on the head-only PRs to be honest - I thought they were only passing because we didn't test ` head ` builds due to the whole deeply unpredictable nature of them, but it's been a fair while since I submitted anything to that repo. I'll try and find an open PR with a recent test to double check.